include/deal.II/numerics/vector_tools_integrate_difference.templates.h

```diff
@@ -1252,17 +1248,19 @@ namespace VectorTools
            ExcMessage("input vector cell_error has invalid size!"));
 #ifdef DEBUG
     {
+      InVector cellwise_error_copy = cellwise_error;
+
+      for (const auto &cell : tria.active_cell_iterators())
+        if (cell->is_locally_owned())
+          cellwise_error_copy[cell->active_cell_index()] = 0.0;
```

Comment:
I don't get this change here. The previous code did not need the copy of the vector -- why was that change necessary?